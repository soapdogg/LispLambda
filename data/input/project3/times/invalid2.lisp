(TIMES 23 4 1)