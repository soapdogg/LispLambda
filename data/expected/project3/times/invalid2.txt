Error! Expected length of TIMES list is 3!    Actual: 4
